docs/proposed/psa-driver-interface.md

```diff
@@ -673,6 +696,14 @@ ECC key pairs are represented as the private key value only. The public key need
 
 The specification doesn't mention when the public key might be calculated. The core may calculate it on creation, on demand, or anything in between. Opaque drivers have a choice of storing the public key in the key context or calculating it on demand and can convey whether the core should store the public key with the `"store_public_key"` property. Is this good enough or should the specification include non-functional requirements?
 
+#### Symmetric key validation with transparent drivers
+
+Should the entry point be called for symmetric keys as well?
+
+#### Key validation with opaque drivers
+
+Are there mandatory validations that an opaque driver must perform on import?
```

Comment:
Wouldn't it suffice to define that when key import returns SUCCESS on an opaque driver, valid operations with said key should succeed accordingly?

Reply:
I defined it as both guaranteeing operation success and the same minimum requirements as `validate_key`, to mandate a baseline (especially for ECDH where accepting an invalid public key can compromise the private key).